imagebuildah/executor.go

```diff
@@ -845,23 +838,18 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (image
 							fields := strings.Split(mountFlags, ",")
 							for _, field := range fields {
 								if strings.HasPrefix(field, "from=") {
-									fromField := strings.SplitN(field, "=", 2)
-									if len(fromField) > 1 {
```

Comment:
I removed this condition for 2 reasons:
1. it can never happen, since we know `=` appears in `field`
2. the error message in the dead else path would log a nil as `formField[1]` would be nil